system/core.go

```diff
@@ -179,7 +186,8 @@ func WriteAPIResponse(resp *APIResponse, code int, w http.ResponseWriter) {
 // http.HanderFunc.
 func APIInvalidHandler(w http.ResponseWriter, _ *http.Request) {
 	resp := &APIResponse{Status: "error",
-		Message: "invalid API version",
+		Code:    codes.InvalidArgument,
```

Comment:
Well that's a catch all right now for invalid version OR invalid command so I think InvalidArgument makes more sense.